src/components/CreateSighting.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react"; 
+import { useState } from "react";
+import { FormEvent } from "react";
+import './CreateSighting.scss';
+
+export function CreateSighting() {
+    const [date, setDate] = useState<string>("");
+    const [photoUrl, setPhotoUrl] = useState<string>("");
+    const [latitude, setLatitude] = useState<number>(0);
+    const [longitude, setLongitutde] = useState<number>(0);
+    const [numberOfWhales, setNumberOfWhales] = useState<number>(0);
+    const [species, setSpecies] = useState<string>("");
+    const [description, setDescription] = useState<string>("");
+
+    function handleSubmit(event: FormEvent<HTMLFormElement>) {
+        event.preventDefault()
+    }
+
+    return (
+        <main className="create-sighting">
+            <h1 className="sighting-form">Post your whale sighting!</h1>
+            <form className="create-sighting-form" onSubmit={(e) => {
+                handleSubmit(e)
+            }
+            }>
```

Comment:
As this function is only one line long we can move it to a single line, something like this might be better to make it easier to read
```suggestion
            <form className="create-sighting-form" 
                  onSubmit={(e) => handleSubmit(e)}>
```